demo/docker-compose.yml

```diff
@@ -101,7 +101,7 @@ services:
             # To use Verification Service in the Cloud change RENDEZVOUS_VERIFICATIONSERVICEHOST variable to point to proper address and set PROXY for SDO service if needed
             # Do not set http_proxy and https_proxy if you use local Verification Service
             # Non-production environment cloud verification sevice:
-            RENDEZVOUS_VERIFICATIONSERVICEHOST: https://verify.epid-sbx.trustedservices.intel.com:443
+            RENDEZVOUS_VERIFICATIONSERVICEHOST: https://verify.epid-sbx.trustedservices.intel.com
```

Comment:
This seems to be an unrelated change. What is the reason for this change?

Reply:
Based on the URL (i.e.) HTTP / HTTPS respective port (80 / 443) is picked up automatically. No need to specify it explicitly.